.github/workflows/pr_approval.yml

```diff
@@ -1,25 +1,34 @@
+# This workflow checks that the PR has been approved by 2+  members of the committee listed in `pull_requests.toml`.
+#
+# Run this pull request when a request review is submitted / dismissed.
+# Note that an approval can be dismissed, and this can affect the job status.
+# We currently trust that contributors won't make significant changes to their PRs after approval, so we accept
+# changes after approval.
```

Comment:
Should we really be making this change in behaviour as part of this PR?

Reply:
This is already the case. That said, I'll remove the pull request trigger. I forgot that it just creates a duplicated workflow